module/testing.py

```diff
@@ -27,10 +28,32 @@ def lock_path(display):
     return '/tmp/.X%d-lock' % display
 
 
+def get_lock(display):
+    lockfile = lock_path(display)
+    if os.path.exists(lockfile):
+        return False
```

Comment:
I think I should remove this check to avoid permanent locking (at least until next reboot) on a certain display if the file was not removed.

Reply:
Done.

Reply:
Yep, agreed.